src/components/Checkout/SocialShare.vue

```diff
@@ -200,13 +204,19 @@ export default {
 		shareMessage() {
 			return this.message.trim() || this.suggestedMessage;
 		},
+		utmContent() {
+			if (this.isGuest) return 'guest';
+			if (this.lender?.public) return this.lender?.inviterName;
```

Comment:
Do we need to also check if `this.lender?.inviterName` is defined here?

Reply:
We could check that. That value is always defined in the database, but there's a chance that an error could occur while trying to resolve it.